src/ape/managers/project/manager.py

```diff
@@ -246,13 +246,11 @@ def _project(self) -> ProjectAPI:
                 self.path, self.contracts_folder
             )
 
-        project = self._cached_projects[self.path.name]
-        project.path = self.path
```

Comment:
Is this still getting set somewhere else? Is this an unnecessary update?

Reply:
It gets set the first, I don't know why it was getting overridden here, but it causes issues.